### PR TITLE
PUB-875 - Filter hide functionality in small screen

### PR DESCRIPTION
**Pull request by marcogagliardi**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-875

### Change description ###
hide filter function in small screen
image

Screenshot 2022-01-27 at 14 37 20

### CODE REVIEW

**Review by furnivall — DISMISSED** [review]
Looks good to me :)

**Comment by kurt-uk** [comment]
> ### JIRA link
> https://tools.hmcts.net/jira/browse/PUB-875
> 
> ### Change description
> hide filter function in small screen Screenshot 2022-01-27 at 14 37 47 Screenshot 2022-01-27 at 14 37 20

is this how it actually looks or have you just not updated webpack?

**Comment by marcogagliardi** [comment]
> > ### JIRA link
> > https://tools.hmcts.net/jira/browse/PUB-875
> > ### Change description
> > hide filter function in small screen Screenshot 2022-01-27 at 14 37 47 Screenshot 2022-01-27 at 14 37 20
> 
> is this how it actually looks or have you just not updated webpack?

it actually looks like that

**Review by furnivall — APPROVED** [review]
👍

**Review by alberto-hortiguela — APPROVED** [review]

**Comment by ChrisS1512** [comment]
Hey. The Hide / Show filters button doesn't appear to be showing for me.

image

I feel like it might be something to do with this: <div class="moj-button-menu filters__innerButton" id="innerButton">, as anything I put in there doesn't appear.

File: src/test/e2e/tests/specs.test.ts [hunk 1]
```diff
@@ -313,69 +264,48 @@ describe('Verified user', () => {
...
+  describe('remove subscription', async () => {
+    before(async () => {
+      await subscriptionManagementPage.open('subscription-management');
+    });
 
-      it('should click continue to create subscription', async () => {
-        pendingSubscriptionsPage = await caseReferenceNumberSearchResultPage.clickContinue();
-        expect(await pendingSubscriptionsPage.getPageTitle()).toEqual('Confirm your subscriptions');
-      });
+    it('should click on the first unsubscribe record', async () => {
```
Comment (ChrisS1512) [inline src/test/e2e/tests/specs.test.ts:275]: Looks like there might be something strange with the subscriptions tests here. Don't believe these should have changed in this ticket so might be a merge conflict issue?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Comment by marcogagliardi** [comment]
> The Hide / Show filters button doesn't appear to be showing for me.
> 
> image
> 
> I feel like it might be something to do with this:
> 
> , as anything I put in there doesn't appear.

The hide/show button will be show only in mobile screen

**Review by ChrisS1512 — APPROVED** [review]

**Review by alberto-hortiguela — APPROVED** [review]

**Review by alberto-hortiguela — DISMISSED** [review]

**Review by ChrisS1512 — DISMISSED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**